app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -36,11 +37,31 @@ class CustomColorViewModel : ViewModel() {
         currentBitmap = b
     }
 
+    var isDialogOpen = false
+    var layerName: String? = null
+    private var layerNumber: Int = 1
+
+    fun makeLayerNameHint(s: String): String {
+        return "$s $layerNumber"
+    }
+
+    fun addLayer(s: String) {
+        if (s.length > NAME_LENGTH)
+            if (s == ("Layer $layerNumber")) {
```

Comment:
This should be in a string resource. Look up formattable string resources

Reply:
Created a formatted string in the Strings XML.